### PR TITLE
Tanner/update basic info

### PR DESCRIPTION
**Pull request by tanner-white**
We can now scroll to reviews by clicking the see all reviews button, and the current product name is now passed up as state to the main app. I also fixed some slight styling issues.

### CODE REVIEW

File: src/overview/components/AddToCart.jsx [hunk 1]
```diff
@@ -29,10 +25,18 @@ function AddToCart({ current }) {
       }
     });
   }, [size, currentSizeQty]);
+
+  const handleClick = (e) => {
```
Comment (Jacob-Gramer) [inline src/overview/components/AddToCart.jsx:29]: Please remove this function if it will not be used

Comment (Jacob-Gramer) [inline src/overview/components/AddToCart.jsx:29]: Please remove function handleClick on line 29 if it is not being used

Comment (tanner-white) [reply src/overview/components/AddToCart.jsx]: Done!

File: src/overview/components/BasicInfo.jsx [hunk 2]
```diff
@@ -3,6 +3,13 @@ import PropTypes from 'prop-types';
 import StarRatings from 'react-star-ratings';
 
 function BasicInfo({ product }) {
+  function handleReviewClick(e) {
+    e.preventDefault();
+    window.scrollTo({
+      top: 1360,
```
[the remark this answers is not in the document]
Comment (tanner-white) [reply src/overview/components/BasicInfo.jsx:11]: Good call, didn't even think about Q&A expanding

**Review by Jacob-Gramer — APPROVED** [review]

**Pull request merged**